core/src/modules/ops/record.ts

```diff
@@ -260,7 +286,8 @@ export namespace RecordOps {
       recordId: string;
       position: number;
       rawValue: string;
-      invalidValue?: string;
+      invalidValue?: string | null;
+      invalidReason?: string | null;
```

Comment:
We assign these things to null a lot. Doing this will help once we turn on strict mode.

Reply:
Hmm, yeah. Reverting this.